pages/index.tsx

```diff
@@ -15,6 +15,7 @@ import UseFormControl from "snet-ui/Notification";
 import Falsemessage from "snet-ui/Flasemessage";
 import Ecosystem from "snet-ui/Ecosystem";
 import Airdropinfo from "snet-ui/Airdropinfo";
+import ColoredLine from "snet-ui/Airdroprules";
```

Comment:
why it is named as `ColoredLine` ?
We should always use descriptive names for all the variables and the components.